src/components/SidebarItem/SidebarItem.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import PropTypes from 'prop-types';
+import { iconUser } from '@wfp/icons';
+import Icon from '../../components/Icon';
+import Tag from '../../components/Tag';
+
+function SidebarItem({
+    image, 
+    title,
+    content,
+    subContent,
+    additionalInfo,
+    noImage,
+    hintInfo,
+    ...other
+}) {
+
+    var profile_image;
+    if(image){
+        profile_image =(
+            <img alt="img" src={image} /> 
+        )
+    }else{
+            profile_image = (<Icon
+                icon={iconUser}
+                fill="#ffffff"
+                width="14"
+                height="14"
+                className="wfp--profile-image--empty"
+                {...other}
+            />)
+    }
+    
+
+    return (
+        <div className="wfp--sidebar-item-wrapper" {...other}>
+            <div className="wfp--sidebar-item__content">   
+            { !noImage ? 
+            (<div className="wfp--profile-image">
+                {profile_image} 
+            </div> )
+            : '' }
+            <div className="wfp--sidebar-content-wrapper">
+                {title && <div className="wfp--item-title">{title}</div>}
```

Comment:
In the future we can replace the `wfp--` prefix with a variable. For example `components/Link/Link.js` is using a prefix variable.

Reply:
sure, can change that